src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java

```diff
@@ -121,6 +122,10 @@ public String getIssueUrl(String issueKey) {
         return qualityGate.getStatus();
     }
 
+    public Map<String,String> getScannerProperties() {
+        return scannerContext.getProperties();
+    }
+
```

Comment:
This only seems to have been introduced to allow logging of all the properties, which doesn't seem like a good use case. The logging should either happen in the PostAnalysisTask that triggers decoration, or be removed.